### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by AndrewUlmer** [graviraja/pytorch-sample-codes#6]
I'm pretty sure the signs on your KLD loss term are flipped: check out the first summation in equation 43 [here](https://arxiv.org/pdf/1907.08956.pdf)
